models/models.js

```diff
@@ -1,5 +1,27 @@
 const db = require('../db/connection');
+const fs = require('fs/promises');
+
+exports.selectApis = () => {
+    return fs.readFile('./endpoints.json', 'utf-8')
+        .then((data) => {
+            const parsedData = JSON.parse(data);
+            return parsedData;
+        });
+};
 
 exports.retrieveTopics = () => {
     return db.query("SELECT * FROM topics;");
-};
\ No newline at end of file
+};
+
+exports.readAllApis = () => {
+    return fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
```

Comment:
Do you really need to read a json to a string, parse it back to json to send it to the client?